wled00/wled_server.cpp

```diff
@@ -520,8 +520,13 @@ void serveSettingsJS(AsyncWebServerRequest* request)
     request->send(401, "application/javascript", buf);
     return;
   }
-  strcat_P(buf,PSTR("function GetV(){var d=document;"));
-  getSettingsJS(subPage, buf+strlen(buf));  // this may overflow by 35bytes!!!
+  byte subSettings = request->arg(F("s")).toInt();
```

Comment:
This is completely necessary if you don't want to break the others settings pages.

Because on the webpage the networks are created dynamically, you cannot set them with GetV, because it'll fail with "unknown element CS{number}", but because we don't know the number of saved network before calling GetV, we cannot create them beforehand.

Reply:
The same principle should be used as in LED settings for adding Color Order Overrides or LED outputs.

You should dynamically generate the number of networks within JS.

Reply:
Implemented in [4a83c0d8dd917af47d42be1838dfcf60b2a7ceb1](https://github.com/Aircoookie/WLED/pull/3705/commits/4a83c0d8dd917af47d42be1838dfcf60b2a7ceb1)